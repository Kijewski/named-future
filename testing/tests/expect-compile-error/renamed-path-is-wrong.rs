use futures_lite::future::block_on;
use named_future::named_future;

#[named_future(crate = some::path)]
async fn answer() -> usize {
    42
}

fn main() {
    let future = answer();
    assert_eq!(42, block_on(future));
}
