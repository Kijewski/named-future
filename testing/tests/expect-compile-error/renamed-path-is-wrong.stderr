error[E0433]: failed to resolve: use of undeclared crate or module `some`
 --> tests/expect-compile-error/renamed-path-is-wrong.rs:4:24
  |
4 | #[named_future(crate = some::path)]
  |                        ^^^^ use of undeclared crate or module `some`
